dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/impl/AsyncExecutor.java

```diff
@@ -246,14 +246,14 @@ private void runExecutionAttempt(AsyncAttemptContextImpl<W> attemptContext, Bulk
             }
 
             if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
-                Tr.event(tc, "Method {0} attempt execution reuslt: {1}", executionContext.getMethod(), methodResult);
+                Tr.event(tc, "Execution {0} attempt execution reuslt: {1}", executionContext.getId(), methodResult);
```

Comment:
"reuslt"

Reply:
Also, do you need to say "execution" twice? Maybe just "Execution {0} result: {1}"?